### ISSUE TITLE
Set the badge count (iOS) to any number 

### ISSUE THREAD

**Issue by webeditors** [pushy-me/pushy-cordova#26]
I have no idea how to do pull requests on GitHub, but came across this problem and figured out how to do it so maybe it could be included in a future release?

My issue and how I made it work is here: [https://stackoverflow.com/questions/63582732/changing-badge-notification-count-with-pushy-in-capacitor-cordova-on-ios](https://stackoverflow.com/questions/63582732/changing-badge-notification-count-with-pushy-in-capacitor-cordova-on-ios)

Basically I tweaked the Pushy swift code and JS to clone the clearBadge function to create a new setBadge function that accepts an integer to change the badge counter to.

**Comment by pushy** [comment]
Hi @webeditors,
Thanks so much for the issue report and feature request, and for the proposed solution.

We find that most clients are using the [Send Notifications API](https://pushy.me/docs/api/send-notifications) to set the badge number:

```
{
    "to": "a6345d0278adc55d3474f5",
    "data": {
        "message": "Hello World!"
    },
    "notification": {
        "body": "Hello World \u270c",
        "badge": 3, <---------------------
        "sound": "ping.aiff"
    }
}
```

And subsequently when the app is opened and notification is handled, the `Pushy.clearBadge()` method is invoked to clear the badge. We may consider adding a `Pushy.setBadge(int)` method in the future if demand from our clients increases.

**Comment by webeditors** [comment]
Yup - I use the Send Notification when I want to send a notification to someone else - it works great - but if I'm on the device I want to  update the count for (ie, if I've read one message but left two unread) I don't really want to send a notification for that; I just want to update it locally. Plus it seems to work whether the user has accepted Push Notifications or not (not tested that intensively yet, but a bonus if that's the case).

Thanks

**Comment by ProximaHaiz** [comment]
@pushy-me Hello. Have you add an ability to increase badge number manually? ( like Pushy.setBadge(int) or etc.)

**Comment by pushy** [comment]
Hi @ProximaHaiz and @webeditors,
Great news, we've just released version `1.0.40` of the Pushy Cordova SDK which adds a new method `Pushy.setBadge(Int)` which can be used as follows in your Cordova project:

```
Pushy.setBadge(3);
```

To be able to access this new method, please update the Pushy Cordova SDK in your app by running the following commands in the root of your Cordova project:

```
cordova plugin remove pushy-cordova-receiver
cordova plugin remove pushy-cordova

cordova plugin add pushy-cordova@1.0.40
cordova plugin add ./plugins/pushy-cordova/receiver/
```

Thanks for your patience, and please let us know if you face any issues.

**Comment by ProximaHaiz** [comment]
@pushy-me Thank you! It is really good news! I will try it on the next week
